experiments/ClimaEarth/run_amip.jl

```diff
@@ -275,6 +275,16 @@ if mode_name == "amip"
         energy_check = energy_check,
     )
 
+    # Update land surface temperature to account for orography
```

Comment:
can we move all of this into `bucket_init`? it would be nice to have it isolated there. maybe we can just pass in `surface_elevation` as an additional argument now

Reply:
Moved to `bucket_init`